crates/router/src/core/payments/tokenization.rs

```diff
@@ -737,9 +742,36 @@ where
             } else {
                 None
             };
-            Ok((pm_id, pm_status))
+            let cmid_config = db
+                .find_config_by_key_unwrap_or(
+                    format!("{}_should_show_connector_mandate_id", merchant_account.get_id().get_string_repr().to_owned()).as_str(),
```

Comment:
Can we instead name this along the lines of `{merchant_id}_connector_mandate_id_in_payments_response`?

Since this indicates whether we're including `connector_mandate_id` in payments response...